asset/css/search-editor.less

```diff
@@ -134,14 +114,6 @@
 
 // Layout
 
-.search-editor-opener {
-  padding: 0 .5em;
-
-  & + a.modal-opener {
-    display: none;
-  }
```

Comment:
Don't remove that. That anchor is added by JS and must not be visible to the user, ever.

Reply:
✅